services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -96,8 +96,14 @@ public void deleteProject(MigrationProject migrationProject)
     }
 
     @Override
+    public void deleteOldProvisionalProjects()
+    {
+        this.migrationProjectService.deleteOldProvisionalProjects(0);
+    }
+
+    @Override
     public Long getProjectIdByName(String title){
-        String jql = "SELECT id FROM MigrationProject p WHERE LOWER(p.title) = LOWER(:title)";
+        String jql = "SELECT id FROM MigrationProject p WHERE LOWER(p.title) = LOWER(:title) AND p.provisional = FALSE";
```

Comment:
IMO it can work as is now. There is no problem in having 2 provisional projects with the same name.